databricks/koalas/series.py

```diff
@@ -5189,6 +5189,100 @@ def explode(self) -> "Series":
         internal = internal.copy(spark_frame=internal.spark_frame.drop(NATURAL_ORDER_COLUMN_NAME))
         return first_series(DataFrame(internal))
 
+    def argmax(self):
+        """
+        Return int position of the largest value in the Series.
+
+        If the maximum is achieved in multiple locations,
+        the first row position is returned.
+
+        Returns
+        -------
+        int
+            Row position of the maximum value.
+
+        Examples
+        --------
+        Consider dataset containing cereal calories
+
+        >>> s = ks.Series({'Corn Flakes': 100.0, 'Almond Delight': 110.0,
+        ...                'Cinnamon Toast Crunch': 120.0, 'Cocoa Puff': 110.0})
+        >>> s  # doctest: +SKIP
+        Corn Flakes              100.0
+        Almond Delight           110.0
+        Cinnamon Toast Crunch    120.0
+        Cocoa Puff               110.0
+        dtype: float64
+
+        >>> s.argmax()  # doctest: +SKIP
+        2
+        """
+        if self.empty:
+            raise ValueError("attempt to get argmax of an empty sequence")
+
+        sdf = self._internal.spark_frame.select(self._internal.data_spark_columns)
+
+        # We should remember the natural sequence started from 0
+        seq_col_name = verify_temp_column_name(sdf, "__distributed_sequence_column__")
+        sdf = InternalFrame.attach_distributed_sequence_column(sdf, seq_col_name)
+
+        col_name = (
+            SPARK_DEFAULT_SERIES_NAME if self._column_label is None else self._column_label[0]
+        )
```

Comment:
I don't think this is a right column name?

Reply:
yeah I just realized that It'd better to use column rather than column name as much as possible in any case.. 😂